deployments/_deployments_log_file.json

```diff
@@ -29722,5 +29722,21 @@
         ]
       }
     }
+  },
+  "GlacisFacet": {
+    "arbitrum": {
+      "staging": {
+        "1.0.0": [
+          {
+            "ADDRESS": "0xF82830B952Bc60b93206FA22f1cD4770cedb2840",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2025-02-26 15:22:09",
+            "CONSTRUCTOR_ARGS": "0x000000000000000000000000d9e7f6f7dc7517678127d84dbf0f0b4477de14e0",
+            "SALT": "",
+            "VERIFIED": "true"
```

Comment:
this is not true, the contract is not verified

Reply:
interesting... Updated.
I didn't touch it manually. Probably there is some bug in the script that mark this contract as verified even if it's not. IMO would be worth to investigate this issue.